packages/editor/src/store/selectors.js

```diff
@@ -485,18 +489,29 @@ export function isEditedPostEmpty( state ) {
 /**
  * Returns true if the post can be autosaved, or false otherwise.
  *
- * @param  {Object}  state Global application state.
+ * @param  {Object} state    Global application state.
```

Comment:
The spacing alignment prior to `{` to align to the `@return` is not necessary or correct, per JavaScript Documentation Guidelines:

https://make.wordpress.org/core/handbook/best-practices/inline-documentation-standards/javascript/#functions

Reply:
Fixed